src/components/box/box.component.tsx

```diff
@@ -45,6 +49,8 @@ export interface BoxProps
   columnGap?: Gap;
   /** Row gap an integer multiplier of the base spacing constant (8px) or any valid CSS string." */
   rowGap?: Gap;
+  /** Box shadow */
+  boxShadow?: BoxShadowsType;
 }
 
 const Box = styled.div<BoxProps>`
```

Comment:
```suggestion
export const Box = styled.div<BoxProps>`
```
Components have to be exported as named exports because storybook has issues with generating propTables otherwise

Reply:
Done as you suggested